scipy/stats/_qmc.py

```diff
@@ -930,10 +843,8 @@ def __init__(self, d, centered=False, seed=None):
 
         # This can be removed once numpy 1.16 is dropped
         try:
-            self.rg_integers = self.rng.randint
             self.rg_sample = self.rng.random_sample
         except AttributeError:
-            self.rg_integers = self.rng.integers
             self.rg_sample = self.rng.random
```

Comment:
Doesn't `rng.random()` work in every version? (Or if I'm forgetting something, `rng.uniform`?)

Reply:
`rng.uniform()` works with `RandomState` and `Random.Generator`.

Reply:
I think `random` does too - just not `rand`.

Reply:
`random` is not available with `RandomState` in numpy <1.17. But yes this could be replaced by `rng.uniform` which works with both methods 👍

Reply:
Thank you for pointing this out! I replaced them with `rng.uniform`.